api.md

```diff
@@ -1,7 +1,14 @@
 # Finch
 
+Types:
```

Comment:
Can we put this under a dedicated section called Request Forwarding ?

Longer term we may want to restructure some of these methods into a section called Management, but we can do that in the next major version upgrade. For now, let's add Request Forwarding under the Webhooks section.